### PR TITLE
4559 Setup CICD

### PR DESCRIPTION
**Pull request by sophiemachin**

### CODE REVIEW

**Comment by this-andy** [comment]
https://this.tpondemand.com/entity/4559

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 paramiko==2.12.0
 https://github.com/THIS-Institute/thiscovery-lib/archive/master.zip
-https://github.com/THIS-Institute/thiscovery-dev-tools/archive/master.zip
+https://github.com/THIS-Institute/thiscovery-dev-tools/archive/refs/heads/sophie-CAPABILITY_NAMED_IAM.zip
```
Comment (this-andy) [inline requirements.txt:3]: Is this right?  It seems odd to have a developer-specific entry in the requirements file.

**Review by this-andy — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (sophiemachin) [reply requirements.txt:3]: Oh good spot, I need to update dev tools to be able to handle this properly

Comment (sophiemachin) [reply requirements.txt]: It turns out I didn't need the custom capability after all. I've put back the requirements.txt, so this can be be reviewed.

**Review by this-andy — APPROVED** [review]

**Pull request merged**